core/silkworm/consensus/validation.hpp

```diff
@@ -64,29 +56,17 @@ enum class [[nodiscard]] ValidationResult{
 
     // See [YP] Section 11.2 "Transaction Validation", Eq (160)
     kWrongBlockGas = 27,  // BHg ≠ l(BR)u
-
     kInvalidSignature = 28,  // EIP-2
-
     kWrongChainId = 29,  // EIP-155
-
-    kUnsupportedTransactionType = 30,  // EIP-2718
+    kInvalidVotingSegment = 30, // Invalid voting segment in Clique/Aura
```

Comment:
kInvalidVotingSegment is never returned. Is corresponding validation not implemented yet or is it a mistake?